ui/src/views/Hooks/ListHooks/index.jsx

```diff
@@ -46,25 +48,80 @@ export default class ListHooks extends Component {
     });
   };
 
+  updateExpanded() {
+    this.query = qs.parse(this.props.location.search.slice(1));
+    const {
+      data: { hookGroups },
+    } = this.props;
+    const hookSearch = this.query.search;
+    const expandedHookGroups = hookGroups
+      ? hookGroups
+          .filter(
+            group =>
+              group.hooks.filter(hook => hook.hookId.includes(hookSearch))
+                .length !== 0
+          )
+          .map(group => group.hookGroupId)
+      : [];
+
+    this.setState({ expanded: expandedHookGroups, selected: [] });
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = { expanded: [], selected: [] };
+    this.updateExpanded();
+  }
```

Comment:
The constructor should be moved to the top of the class (function).

Reply:
Moved all the way up

Reply:
Great :) I'm still reviewing this pr - I'll let you know when I'm done so you can batch the changes together.

Reply:
Oj, sorry I thought you made all the comments in batch "review" feature, my bad.

Reply:
No worries :)